src/schema/rules/files/raw/channels.yaml

```diff
@@ -16,6 +16,16 @@ channels:
     acquisition: optional
     run: optional
 
+# ephys has one removed
+# TODO: see if to be added for consistency etc.
+channels__ephys:
+  $ref: rules.files.raw.channels.channels
+  datatypes:
+    - meg
+  entities:
+    $ref: rules.files.raw.channels.channels.entities
+    acquisition:
+
```

Comment:
here is an initial motivator for adding `_acq-` entity.  WDYT @bids-standard/bep032 ppl -- could there be an _acq entity which is quite commonly used ... Actually looking at https://bids-specification.readthedocs.io/en/stable/appendices/entity-table.html -- it is used EVERYWHERE besides PET, so as to me it is a strong indicator to add it to ephys and related files as well.

Reply:
@yarikoptic I think this comment should probably be on the google doc at this point. Having `_acq` seems fine. Do you have an idea of what specific types of information would be encoded in this? Do you have ideas of examples where this would be useful?

Reply:
yes -- let's discuss/decide in the google doc. It would be useful for annotating any differently acquired data in the same session, e.g. different probes, equipment, etc. 

here I left it empty as to signal "remove" (breaks the schema actually) per "google doc" since inherited `rules.files.raw.channels.channels.entities` already has it.

Reply:
indeed, my question would be: in the context of ephys, how would we use the '_acq' entity?

Reply:
as suggested above and in line with purpose of `_acq` in any other modality -- "free" (there is no controlled vocabulary) annotation of any difference in acquisition between two files which otherwise can be named the same, e.g. same task, but different hardware - e.g. `_acq-neuropixel` and `_acq-spikeglx`  if someone decided to do 1-to-1 comparison recording data from the same neuropixel probe using two different support acquisition systems?

Reply:
@SylvainTakerkart the genealogy of `_acq` was MR acquisition protocols, but it has morphed into general-purpose categorization of acquisition. It's fairly useful for “miscellaneous” information. I'd also be for keeping it in, and maybe including it for PET as well.

Reply:
ok, decided -- we will have `_acq` here. PET would be another story for another PR ;)